### DISCUSSION TITLE
Documentation - Features list

### DISCUSSION THREAD

**Discussion by dominiclapointe** [oqtane/oqtane.framework#838] [General] [answered]
Hi,

I would like to know if there is any documentation regarding usage of octane and if a specific features list is available somewhere? Basically, i am trying to compare abp.io blazor and octane as a base starter kit for a new project but i cant find clear documentation not features list for octane as abp.io do expose here : https://docs.abp.io/en/abp/latest

Thank you

**Comment by hishamco** [comment]
@sbwalker  can we move all the docs issues into the docs repo

**Comment by sbwalker** [comment, marked as answer]
@dominiclapointe there is quite a bit of information on the home page of the repo: https://github.com/oqtane/oqtane.framework

Current Feature Set:

Multi-Tenant ( Shared Database & Isolated Database )
Modular Architecture
Headless API with Swagger Support
Dynamic Page Compositing Model / Site & Page Management
Authentication / User Management / Profile Management
Authorization / Roles Management / Granular Permissions
Dynamic Routing
Extensibility via Custom Modules
Extensibility via Custom Themes
Event Logging
Folder / File Management
Recycle Bin
Scheduled Jobs ( Background Processing )
Notifications / Email Delivery
Auto-Upgrade Framework
Progressive Web Application Support

Next Release - V2.0 - Nov 10, 2020

.NET 5 Support
Static Localization

Video Tutorials: https://www.youtube.com/watch?v=JPfUZPlRRCE&list=PLYhXmd7yV0elLNLfQwZBUlM7ZSMYPTZ_f
( you should watch the first video in the series as it talks about the difference between a modular framework and boilerplate apps)

Documentation: https://docs.oqtane.org/api/index.html
( this is a starting point and we hope the community will provide more content )

**Reply by dominiclapointe** [reply]
Thank you @sbwalker. The actual documentation is quite interesting. Where did you get that link? is it not yet published or did i completly missed it?

**Reply by sbwalker** [reply]
@dominiclapointe a community member (@iJungleboy) has been working on the configuration to auto generate API documentation using Docfx and the domain was only wired up last week. We definitely need to promote this more broadly.

**Reply by dominiclapointe** [reply]
Just want to add : congrats to you (@sbwalker and the contributors)! i have been watching your video at C# Corner and oqtane is really promising and well designed! 

Unfortunately it doesn't fit our needs for the actual project but will most definitively try to get onboard for a future project and will keep following you!
